src/dali_model_instance.h

```diff
@@ -94,12 +99,17 @@ class DaliModelInstance : public ::triton::backend::BackendModelInstance {
   ProcessingMeta ProcessRequests(const std::vector<TritonRequest>& requests,
                                  const std::vector<TritonResponse>& responses);
 
+  TimeInterval ProcessRequest(const TritonRequest &request);
+
```

Comment:
Why is there no `responses` in the signature?

Reply:
ProcessRequest creates and send the responses - we don't know how many responses will be there for a request, so I cannot create them  before processing the request.